modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/multiselect/MultiSelectBookmarksHelper.kt

```diff
@@ -35,6 +36,7 @@ class MultiSelectBookmarksHelper @Inject constructor(
 
     override val toolbarActions: LiveData<List<MultiSelectAction>> = _selectedListLive
         .map {
+            Sentry.addBreadcrumb("MultiSelectBookmarksHelper toolbarActions updated, ${it.size} bookmarks from $source")
```

Comment:
Since bookmarks haven't been released, this shouldn't ever fire, but I added this breadcrumb anyway just so we can confirm that.